images/tensorflow/Dockerfile

```diff
@@ -12,23 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG image_version=nightly
+ARG image_version="nightly"
+
 FROM tensorflow/tensorflow:${image_version}-gpu
 
```

Comment:
Manually install tensorflow-gpu pip package in python:3.9.16-bullseye image.

`RUN pip install tensorflow-gpu`